src/assets/wise5/components/dialogGuidance/DialogGuidanceFeedbackRuleEvaluator.ts

```diff
@@ -114,11 +114,24 @@ export class DialogGuidanceFeedbackRuleEvaluator {
     return !this.evaluateTerm(termStack.pop(), response);
   }
 
-  private evaluateOrExpression(term1: string, term2: string, response: CRaterResponse): boolean {
-    return this.evaluateTerm(term1, response) || this.evaluateTerm(term2, response);
+  private evaluateTerm(term: string, response: CRaterResponse): boolean {
+    if (this.isHasKIScoreTerm(term)) {
+      return this.evaluateHasKIScoreTerm(term, response);
+    } else {
+      return this.evaluateIdeaTerm(term, response);
+    }
   }
 
-  private evaluateTerm(term: string, response: CRaterResponse): boolean {
+  private isHasKIScoreTerm(term: string): boolean {
+    return /hasKIScore\([1-5]\)/.test(term);
+  }
```

Comment:
Should this be /hasKIScore\(\d+\)/ in case we ever have item ids that return scores more than 5?